core-blocks/text-columns/index.js

```diff
@@ -25,6 +26,9 @@ import './editor.scss';
 export const name = 'core/text-columns';
 
 export const settings = {
+	// Use an empty parent array to indicate this block should no longer be available for insertion.
+	parent: [],
```

Comment:
A more "official" way of doing this is to use `supports.inserter: false`. Up to you which way you go—they should be functionally equivalent 🙂

https://github.com/WordPress/gutenberg/blob/7ab5a9e8a6d07bb2d16e21fe2841e4619e64f0c1/core-blocks/block/index.js#L23-L27

Reply:
Let's follow what we have for Shared block 👍

Reply:
That's much more explicit. I'll switch to that. Thanks for letting me know.